solr/bin/solr

```diff
@@ -982,31 +982,10 @@ else
   exit
 fi
 
+# status tool
 if [ "$SCRIPT_CMD" == "status" ]; then
-  if [ $# -gt 0 ]; then
-    while true; do
-      case "$1" in
-          -help|-h)
-              print_usage "$SCRIPT_CMD"
-              exit 0
-          ;;
-          --)
-              shift
-              break
-          ;;
-          *)
-              if [ "$1" != "" ]; then
-                print_usage "$SCRIPT_CMD" "Unrecognized or misplaced argument: $1!"
-                exit 1
-              else
-                break # out-of-args, stop looping
-              fi
-          ;;
-      esac
-    done
-  fi
   get_status
-  exit
+  exit $?
```

Comment:
I like these very much! Makes me almost believe we can get `solr.cmd` right as well 😄

Reply:
Only complaint I have is that usage help print in SolrCLI is not very nice. So I did #2545 in main to Improve on it..